### ISSUE TITLE
3CQS bot banned from Telegram - (caused by SendCodeRequest)

### ISSUE THREAD

**Issue by Gigamax9** [TBMoonwalker/3cqsbot#72]
Good day

I started a 3cqs bot two days ago and up until this evening, everything had been going quite smoothly. Then this evening I examined the config.ini file in greater detail, changing three or so of the settings as I went. I then stopped the bot and restarted it. All seemed fine for around 10 minutes before an error message appeared indicating my number had been banned from Telegram. 

error message: "telethon.errors.rpcerrorlist.PhoneNumberBannedError: The used phone number has been banned from Telegram and cannot be used anymore. Maybe check https://www.telegram.org/faq_spam (caused by SendCodeRequest)"

Two of the settings I changed were:
activated to this: deal_mode = [{"options": {"time": "5m", "type": "buy_or_strong_buy"}, "strategy": "trading_view"},{"options": {"time": "15m", "points": "70", "time_period": "7", "trigger_condition": "less"}, "strategy": "rsi"},{"options": {"time": "1h", "points": "70", "time_period": "7", "trigger_condition": "less"},{"options": {"time": "4h", "points": "70", "time_period": "7", "trigger_condition": "less"}]

activated: btc_pulse = true

Is there someone out there that can perhaps shed some light on this problem?

**Comment by Gigamax9** [comment]
Follow up:
From my phone where a pop-up window appeared indicating my profile had been banned, I sent an email to Telegram explaining the situation.
i.e. that I'd not sent any messages to any chats other than those sent by my bot to its own chat. 
The profile has now been reactivated. I have started a new 3cqs bot session. I changed the deal-mode setting from #multiple strategies back to # single strategy. I suspect this may have caused the spam/block trigger.
